$OpenBSD$

nstime->secs is int maybe should be defined as long?

https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=6738

--- epan/wslua/wslua_pinfo.c.orig	Thu Jan 12 15:10:42 2012
+++ epan/wslua/wslua_pinfo.c	Thu Jan 12 17:43:47 2012
@@ -95,7 +95,7 @@ WSLUA_METAMETHOD NSTime__tostring(lua_State* L) {
 
     if (!nstime) return 0;
 
-    lua_pushstring(L,ep_strdup_printf("%ld.%09d", nstime->secs, nstime->nsecs));
+    lua_pushstring(L,ep_strdup_printf("%ld.%09d", (long) nstime->secs, nstime->nsecs));
 
     WSLUA_RETURN(1); /* The string representing the nstime. */
 }
